libs/entitlement/src/main/java/org/elasticsearch/entitlement/runtime/policy/entitlements/FilesEntitlement.java

```diff
@@ -239,9 +239,7 @@ public Stream<Path> resolveRelativePaths(PathLookup pathLookup) {
                 String path = pathLookup.settingResolver().apply(setting);
                 result = path == null ? Stream.of() : Stream.of(path);
             }
-            if (ignoreUrl) {
-                result = result.filter(s -> s.toLowerCase(Locale.ROOT).startsWith("https://") == false);
-            }
+            result = result.filter(s -> s.toLowerCase(Locale.ROOT).startsWith("https://") == false);
```

Comment:
Should we ignore all URLs? http://, ftp:// (if we even allow that thing).
Even file:// if passed down to FileAccessTree as is would make a mess of the path sting.